app/src/main/java/ch/hikemate/app/utils/LocationUtils.kt

```diff
@@ -245,7 +245,8 @@ object LocationUtils {
           progressDistance = projected.distanceTo(segment.start),
           distanceFromRoute = location.distanceTo(projected),
           segment = segment,
-          indexToSegment = 0)
+          indexToSegment = 0,
+          projectedLocationElevation = route.elevation[0])
```

Comment:
Suggestion: maybe we should handle the case here where `route.elevation` is an empty list.

Reply:
You're right! I changed it to be nullable so that we handle this case!